test/helpers/SessionAnswersHelperSpec.scala

```diff
@@ -1146,4 +1156,53 @@ class SessionAnswersHelperSpec extends SpecBase {
       }
     }
   }
+
+  "getPreviousUploadsFileNames" should{
+    "return Future[String] " in {
+      val mockRepository = mock(classOf[UploadJourneyRepository])
+      implicit val ec: ExecutionContext = injector.instanceOf[ExecutionContext]
+      val fakeRequestForOtherJourney: UserRequest[AnyContent] = fakeRequestConverter(fakeRequestWithCorrectKeys.withSession(
+        SessionKeys.reasonableExcuse -> "other",
+        SessionKeys.hasConfirmedDeclaration -> "true",
+        SessionKeys.whyReturnSubmittedLate -> "This is a reason.",
+        SessionKeys.whenDidBecomeUnable -> "2022-01-02",
+        SessionKeys.journeyId -> "4321"
+      ))
+      val callBackModel: UploadJourney = UploadJourney(
+        reference = "ref1",
+        fileStatus = UploadStatusEnum.READY,
+        downloadUrl = Some("download.file/url"),
+        uploadDetails = Some(UploadDetails(
+          fileName = "file1.txt",
+          fileMimeType = "text/plain",
+          uploadTimestamp = LocalDateTime.of(2018, 1, 1, 1, 1),
+          checksum = "check1234",
+          size = 2
+        ))
+      )
+      when(mockRepository.getUploadsForJourney(Some("4321"))).thenReturn(Future(Option(Seq(callBackModel))))
+      val  helper = new SessionAnswersHelper(mockRepository)
+      whenReady(helper.getPreviousUploadsFileNames()(fakeRequestForOtherJourney)){
+       result => result shouldBe "file1.txt"
+     }
+    }
+  }
+
+  "getContentWithExistingUploadFileNames" should{
+    "return Future[Seq[(String, String, String)]] " in {
+      implicit val ec: ExecutionContext = injector.instanceOf[ExecutionContext]
+      val fakeRequestForOtherJourney: UserRequest[AnyContent] = fakeRequestConverter(fakeRequestWithCorrectKeys.withSession(
+        SessionKeys.reasonableExcuse -> "other",
+        SessionKeys.hasConfirmedDeclaration -> "true",
+        SessionKeys.whyReturnSubmittedLate -> "This is a reason.",
+        SessionKeys.whenDidBecomeUnable -> "2022-01-02",
+        SessionKeys.journeyId -> "4321"
+      ))
+      val mockHelper = mock(classOf[SessionAnswersHelper])
+      val result:Future[Seq[(String, String, String)]] = Future(Seq())
+      when(mockHelper.getContentWithExistingUploadFileNames(Some("other"))(fakeRequestForOtherJourney,messages)).thenReturn(result)
+      mockHelper.getPreviousUploadsFileNames()(fakeRequestForOtherJourney)
+      verify(mockHelper, times(1)).getPreviousUploadsFileNames()(fakeRequestForOtherJourney)
```

Comment:
This isn't really testing much, other than we have called the method

Reply:
Yes, intended to test the behaviour.

Reply:
This is testing the behaviour that was mocked on line 1203, we want it to test that SessionAnswersHelper does it job and returns the correct values